exercises/grade-school/canonical-data.json

```diff
@@ -37,7 +37,11 @@
       "description": "Adding multiple students in the same grade in the roster",
       "property": "add",
       "input": {
-        "students": [["Blair", 2], ["James", 2], ["Paul", 2]]
```

Comment:
had I been the one making this PR, I would have kept the existing formatting, but I have no disagreement with the decision made in this PR.